### DISCUSSION TITLE
The objects of my calls are as mine

### DISCUSSION THREAD

**Discussion by FranGomezVenegas** [classgraph/classgraph#613] [General]
Sorry if the subject is not exciting, I hope the figure below is more descriptive .... let me add some detail about what I'm looking for. The thread is more about know if this is possible before starting going deep inside without certain success level ;)

The lines in yellow are the codes that each logic uses, those are the codes I need to get by reflection, let me introduce you to the example in the figure below.

The orange methods are the typical common methods that any other method can consume.
The blue boxes are the methods that are involved in this UpdateIncubator endpoint. 
The green box (it consumes the oranges but resolve its need without calling any other method) is another endpoint, UpdateOtherEntity.
What I tried to illustrate through the arrows is that the dbUpdateTransaction is called in a different way. The blue box is indirectly called due to it is called by another method. The green box calls it directly.

The goal, is that for the blue box I need to trap all the yellows except the one in the green box.
For the green box I need to trap the yellows in the orange boxes and in itself, in other words, all except the blue boxes ones.

Is that possible? Any constraint to consider? I mean, in this example I go down up to 4 levels (in this one, endpoint >updateIncubator>isIncubatorActive>dbUpdateTransaction)

![image](https://user-images.githubusercontent.com/26067165/146175768-d196b6d2-fe6d-40bb-bbc0-af6ba228d79c.png)

Thanks in advance

**Comment by FranGomezVenegas** [comment]
What shown above is just one example to illustrate one goal, It is not a real way the code is splited in methods but I think is enough representative of typical methods in a RESTFULL API

**Comment by lukehutch** [comment]
@FranGomezVenegas did you post this question to the correct project? What does this have to do with ClassGraph?

**Reply by FranGomezVenegas** [reply]
I think so, I guess. What I'm looking for is to get all methods being invoked by one method (and then the same for those sub-methods) and then get any call to this method called LPPlatform.trapMessage in order to get the enum entry being in use. Is that possible?

**Reply by lukehutch** [reply]
As far as I can tell this has nothing whatsoever to do with the ClassGraph library. If it does, how are you trying to use ClassGraph?
